conf/adcm/python_scripts/run_ldap_sync.py

```diff
@@ -13,31 +13,47 @@
 
 import os
 import sys
+import logging
 import ldap
 
 os.environ["PYTHONPATH"] = "/adcm/python/"
 sys.path.append("/adcm/python/")
 
 import adcm.init_django  # pylint: disable=unused-import
 from rbac.models import User, Group, OriginType
-from rbac.ldap import _get_ldap_default_settings, configure_tls, is_tls
+from rbac.ldap import (
+    configure_tls,
+    get_groups_by_user_dn,
+    get_ldap_config,
+    get_ldap_default_settings,
+    get_user_search,
+    is_tls
+)
 from cm.errors import AdcmEx
 from cm.logger import log
 from django.db import DataError, IntegrityError
 
 CERT_ENV_KEY = "LDAPTLS_CACERT"
 
 
+stdout_log = logging.getLogger("stdout")
```

Comment:
nit: variable contains logger, may be it can be named like `stdout_logger`

Reply:
done